src/com/shapestone/laptopshoppingandbilling/PrintTotalPayment.java

```diff
@@ -0,0 +1,38 @@
+package com.shapestone.laptopshoppingandbilling;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class PrintTotalPayment {
+
+	public void printTotalPayment(ArrayList<Customers> customers, ArrayList<Purchases> purchases) {
+
+		System.out.println(
+				"---------------------------------------------------------------------------------------------------------------------------------------------------------------");
+		System.out.printf("| %-15s | %-17s | %-17s | %-17s | %-17s | %-28s | %-15s | %-9s |%n", "S.NO", "Customer ID",
+				"Name", "Age", "Gender", "Date Of Purchase", "Product ", "Price ");
+		System.out.println(
+				"---------------------------------------------------------------------------------------------------------------------------------------------------------------");
+
+		double totalAmount = 0.0;
+
+		for (int i = 0; i < customers.size(); i++) {
```

Comment:
calculate total amount for each customer

Reply:
ok sir